pyfr/solvers/euler/kernels/flux.mako

```diff
@@ -24,3 +24,40 @@
     f[${i}][${j + 1}] = rhov[${i}]*v[${j}]${' + p' if i == j else ''};
 % endfor
 </%pyfr:macro>
+
+<%pyfr:macro name='inviscid_flux_1d' params='s, f, p, v'>
```

Comment:
Lets move this to the exact solver file as it is only used in one place.

Reply:
I didn't do it here, but if you were to add other 1D Riemann solvers they would need this, hence why I put it here. 
Once this is merged, my plan was to refactor the HLLC and Roe solvers as they are much cleaner when implemented in this format.